README.md

```diff
@@ -1,4 +1,6 @@
-# Content-o-Tron :robot:
+<img src="assets/mark.svg" width="200px">
```

Comment:
I think we could do
```
# Content-o-Tron
<div align=center>
    <img src="assets/mark.svg" width="200px">
</div>
```

Reply:
This doesn't seem to work
<img width="671" alt="screen shot 2018-05-16 at 14 09 45" src="https://user-images.githubusercontent.com/1917629/40116098-db6752d8-5912-11e8-9f5e-5de3e715f8ea.png">

Reply:
I think so it would once you commit. For reference, see [README](https://github.com/channelCS/Audio-Vision/tree/master/DNN)

Reply:
However it can be achieved with

```gfm
<p align="center">
  <img src="assets/mark.svg" width="200px">
</p>
```

To do this:

<img width="949" alt="screen shot 2018-05-16 at 14 11 36" src="https://user-images.githubusercontent.com/1917629/40116196-2e6b48e0-5913-11e8-91a0-56101f5564cf.png">

I however prefer the left one :)

Reply:
We agree that the **heading goes to at the top**. For logo alignment, I think we should see what @booya has to say 😅 Btw, that looks so good 😍

Reply:
Just to check, is Ferrisbot left alighned and above the heading (Content-o-Tron)?

I prefer this, over trying to centre aligning him to the page / README.md

We can worry about positioning when we start on #35 

Thanks again @bvhme for your logo! :D

Reply:
So should he be under or over the heading @booyaa

Reply:
I think he means logo(❤️) should be above the heading and left aligned.